src/components/Body/MovieCards/MovieCards.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import movies from "../../../assets/movies/movies";
+import { Component } from "react";
+import MovieCard from "./MovieCard/MovieCard";
+import { MovieCardsState, MovieCardsProps } from "./movieCards.types";
+import styles from "./movieCadrds.module.css";
+
+class MovieCards extends Component<MovieCardsProps, MovieCardsState> {
+  state: MovieCardsState = {
+    movies: [],
+  };
+
+  componentDidMount() {
+    this.setState({ movies });
+  }
```

Comment:
I have many questions about this component

Why are you using a class component?

Why are you specifying the type of props you are passing if you are not actually passing them?